package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apigeetool",
-  "version": "0.14.2",
```

Comment:
Please undo this change, we will bump the `version` as we package/release the next release.

Reply:
Completed as requested